### ISSUE TITLE
cDock not working on 10.13.2

### ISSUE THREAD

**Issue by peterb202** [w0lfschild/cDock2#82]
Just updated to High Sierra 10.13.2 on a MacBook Pro. Installed cDock 0.12.4. SIP was already disabled for fresh Total Finder install and still is.
cDock starts but blanks out the "theming" tab. "Install" SIMBL button does nothing. Installing SIMBL manually (with SIP off) does not change anything. Reinstalling cDock and  SIMBL does not help. 
Only when logging off and back in Finder says cDockHelper.app was blocked. Trying to unblock it via security control panel does nothing. Trying to start it via terminal seems to crash it (see Console log):

Process:               cDockHelper [1704]
Path:                  /private/var/folders/*/cDockHelper.app/Contents/MacOS/cDockHelper
Identifier:            org.w0lf.cDock-Helper
Version:               0.12.4 (4039)
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           cDockHelper [1704]
User ID:               501

Date/Time:             2017-12-13 21:26:27.508 +0100
OS Version:            Mac OS X 10.13.2 (17C88)
Report Version:        12
Anonymous UUID:        23FEC767-C9BD-3B1A-641E-CAE799CC922A

Sleep/Wake UUID:       6B9BFDA9-606D-4CA3-9BBE-79FC937F933A

Time Awake Since Boot: 4500 seconds
Time Since Wake:       1500 seconds

System Integrity Protection: disabled

Notes:                 Translocated Process

Crashed Thread:        0

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Reason:    DYLD, [0x1] Library missing

Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
  Library not loaded: @rpath/Sparkle.framework/Versions/A/Sparkle
  Referenced from: /private/var/folders/*/cDockHelper.app/Contents/MacOS/cDockHelper
  Reason: image not found

Binary Images:
       0x10ad4c000 -        0x10ad50ff3 +org.w0lf.cDock-Helper (0.12.4 - 4039) <957946B3-3B44-3C86-8CB9-3CD53D00E184> /var/folders/*/cDockHelper.app/Contents/MacOS/cDockHelper
       0x113ef0000 -        0x113f3a98f  dyld (519.2.2) <6695F30B-4E88-3C0B-9867-7D738C44A3E6> /usr/lib/dyld

**Comment by w0lfschild** [comment]
Hmm looks like it has to do with it not finding the sparkle Framework. Not sure why that's happening though?

You could try opening Terminal and running `osascript -e "tell application Dock to inject SIMBL"`

**Comment by peterb202** [comment]
Sorry I somewhat thought I had reported that the terminal-code did not work.

Obviously I did not and so you closed the issue. However cDock still is not working for me.

Pasting the suggested code into terminal brings up a syntax error:
 syntax error: „Identifier“ kann nicht diesem „Identifier“ folgen. (-2740)

-> identifier cannot follow this identifier (-2740)

While I am not much of a coder I am not sure what to do with this.

best regards
hans
